From 127d8677bffa667736713eb1c1cb6378222a1822 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 22 Mar 2024 20:42:47 -0500
Subject: [PATCH] Install the package.xml manifest

---
 rpm/ros-rolling-fastrtps.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/ros-rolling-fastrtps.spec b/rpm/ros-rolling-fastrtps.spec
index 1a8c3cdb2a71..acf607f07e8c 100644
--- a/rpm/ros-rolling-fastrtps.spec
+++ b/rpm/ros-rolling-fastrtps.spec
@@ -87,6 +87,7 @@ internals.
 
 %install
 %cmake_install
+install -m0644 -p -D package.xml %{buildroot}%{bloom_prefix}/share/%{pkg_name}/package.xml
 
 
 %if 0%{?with_tests}
-- 
2.44.0

